### PR TITLE
Allow enable/disable speaker and microphone from UI

### PR DESCRIPTION
**Pull request by Adarsha-gg**
Solves  issuse: #51 with commit: bff4eac59e055bc3a267369caf6f46ccbc0e5538 

Addition of a new UI for enabling disabling microphone/speaker as per need.

### CODE REVIEW

**Comment by vivekuppal** [comment]
Recommend to implement the functionality as Menu options.
Attempting to reduce the number of buttons in the UI. Several of the other buttons have been moved to File Menu as well.
E.g could do Edit -> Speaker Enabled, Edit -> Microphone Enabled

The button click behavior works correctly.

Start with the following cmd line arguments
	- python main.py -ds
	Speaker button should say Speaker Disabled, currently it says speaker enabled
	Similar behavior for microphone for -dm option

**Comment by Adarsha-gg** [comment]
Improvements in commit: [ab7a0b8]

Merged the two functionalities into a single button -> Menu. Using command line arguments doesnt break the button.

Only problem is the arguments code has been moved down in the main.py file to 151.  I couldn't find a way to intialize the variable without altering the code structure a little bit.

**Comment by Adarsha-gg** [comment]
Improvements in commit: [2025a84] 

Removed the "Menu" button and added the enable/disable speaker/microphone to the edit button at top of the application.
It is compatible with system arguments such as -ds and -dm.

**Review by vivekuppal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Run ```generate_binary.bat``` file by replacing paths at the top of the file to
 
 1. Download the zip file from
 ```
-https://drive.google.com/file/d/1UJR4rJt5umxxOiAHNGXPMhnnYr6Bj6fX/view?usp=sharing
```
Comment (vivekuppal) [inline README.md:107]: What are the changes in the binary files

Comment (vivekuppal) [reply README.md:107]: If there is no change to the binaries, the link should stay the same as before.

**Pull request merged**